evm/src/main/java/org/hyperledger/besu/evm/internal/EvmConfiguration.java

```diff
@@ -15,20 +15,19 @@
 
 package org.hyperledger.besu.evm.internal;
 
-/** The Evm configuration. */
-public class EvmConfiguration {
-  /** The constant DEFAULT. */
-  public static final EvmConfiguration DEFAULT = new EvmConfiguration(32_000L);
+import java.nio.file.Path;
+import java.util.Optional;
 
+public class EvmConfiguration {
+  public static final EvmConfiguration DEFAULT = new EvmConfiguration(32_000L, Optional.empty());
   private final long jumpDestCacheWeightKB;
 
-  /**
-   * Instantiates a new Evm configuration.
-   *
-   * @param jumpDestCacheWeightKB the jump dest cache weight kb
-   */
-  public EvmConfiguration(final long jumpDestCacheWeightKB) {
+  private final Optional<Path> kzgTrustedSetupPath;
```

Comment:
The KZG setup is only relevant to some versions of the EVM, so it is not appropriate as a global configuration option.  Treat it like `ChainID` has been treated.